x-pack/plugins/apm/server/index.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import { PluginInitializerContext } from 'src/core/server';
+import { APMOSSConfig } from 'src/plugins/apm_oss/server';
+import { APMPlugin } from './plugin';
+
+export const config = {
+  schema: schema.object({
+    servicemapEnabled: schema.boolean({ defaultValue: false }),
+    bucketTargetCount: schema.number({ defaultValue: 15 }),
+    minimumBucketSize: schema.number({ defaultValue: 15 }),
```

Comment:
`minimumBucketSize` and `bucketTargetCount` are carry-over from the opbeat days (yeah!). 
If you think it makes sense in this PR it would be great if you can just make them constants (like `apmAgentConfigurationIndex`)

Reply:
Moved these to plain old constants in the `server/lib/transactions/constants`. They're not used outside of transactions AFAICT.